Source/WebKit/UIProcess/API/Cocoa/_WKWebsiteDataStoreDelegate.h

```diff
@@ -72,4 +72,5 @@ WK_API_AVAILABLE(macos(10.15), ios(13.0))
 - (void)websiteDataStore:(WKWebsiteDataStore *)dataStore domain:(NSString *)registrableDomain didOpenDomainViaWindowOpen:(NSString *)openedRegistrableDomain withProperty:(WKWindowProxyProperty)property directly:(BOOL)directly;
 - (void)websiteDataStore:(WKWebsiteDataStore *)dataStore didAllowPrivateTokenUsageByThirdPartyForTesting:(BOOL)wasAllowed forResourceURL:(NSURL *)resourceURL;
 - (void)websiteDataStore:(WKWebsiteDataStore *)dataStore domain:(NSString *)registrableDomain didExceedMemoryFootprintThreshold:(size_t)footprint withPageCount:(NSUInteger)pageCount processLifetime:(NSTimeInterval)processLifetime inForeground:(BOOL)inForeground;
+- (NSData *)_webCryptoMasterKeyForWebProcess;
```

Comment:
Is the crypto master key going to change in run time? If not, I think `WKWebsiteDataStoreConfiguration` is a better place for this value, so we can ask client to provide it at the creation time instead of pulling it.

Reply:
We currently only need to access the master key when a page uses crypto APIs.  Making this change would make it so that we always need to access the key from the keychain before we can show any page.